# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfig/mandir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xfig.3.2.5/Imakefile.vanilla	2007-05-09 11:08:11.000000000 +0000
+++ xfig.3.2.5/Imakefile	2007-05-09 11:08:49.000000000 +0000
@@ -175,7 +175,7 @@
 XCOMM MANDIR tells where the standard man pages should go (no need to change it
 XCOMM if you want the man pages installed in the standard place on your system
 XCOMM MANDIR = $(MANSOURCEPATH)$(MANSUFFIX)
-MANDIR = /usr/local/xfig/man
+MANDIR = $(MANPATH)
 
 XCOMM If your system doesn't have strstr undefine the following definition
 XCOMM HAVE_NO_NOSTRSTR = -DNOSTRSTR
